timeline-chart/src/components/time-graph-row-element.ts

```diff
@@ -30,6 +30,7 @@ export class TimeGraphRowElement extends TimeGraphComponent {
     ) {
         super(id, displayObject);
         this._height = _style.height || 14;
+        this._height = _row.height === 0? 0 : Math.min(this._height, _row.height -1);
```

Comment:
Small formatting comments. Please add a space before ? and 1 
        this._height = _row.height === 0 ? 0 : Math.min(this._height, _row.height - 1); 

Sorry, I missed that earlier.

Reply:
Thank you! Done.